kademlia/routing.py

```diff
@@ -1,8 +1,9 @@
 import heapq
 import time
 import operator
-from collections import OrderedDict
+import asyncio
 
+from collections import OrderedDict
```

Comment:
Unnecessary changes should not be committed as part of a pull request.  If you move this line back, then I'll happily merge.  Thanks!